demo-project/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
```

Comment:
Personal preference, do we need this?

Reply:
It's the same option in the frontend, I wanted to keep it consistent.

Reply:
I would remove it from the frontend as well, but don't wanna hang too much on these comments, so I'll resolve it. Feel free to change if you like.